spec/features/user_can_delete_post_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.feature "Delete post", type: :feature do
+  scenario "user can delete own post" do
+    log_in
+    post_picture
+    expect(first('.each-post')).to have_link "Delete"
+    click_link "Delete"
+    expect(page).to_not have_selector('.each-post')
+  end
+
+  scenario "user cannot delete someone else's picture" do
+    user = User.create(username: "TestUser2", email: "Test2@example.com", password: "password")
```

Comment:
Is line 13 already included in the helper log_in method?

Reply:
sorry just realising this is a different user, ignore this

Reply:
no this is the creation of a 2nd user so that I can test that the logged in user cannot access another person's post. I will update it so that the variable is named better and include a comment so that it is easier to understand. 👍